contrib/craftscripts/maze.js

```diff
@@ -23,19 +23,19 @@ importPackage(Packages.com.sk89q.worldedit.blocks);
 usage = "<block> [width] [length] [height] [size] [thickness] flags\n";
 usage += "\n";
 usage += "Maze Flags:\n";
-usage += "• i adds an entry and an exit\n";
-usage += "• y places the entry and the exit randomly\n";
-usage += "• f adds a floor\n";
-usage += "• c adds a ceiling\n";
-usage += "• e places air blocks\n";
-usage += "• a places air blocks only\n";
-usage += "• v creates a vertical maze\n";
+usage += "• i adds an entry and an exit\n";
+usage += "• y places the entry and the exit randomly\n";
+usage += "• f adds a floor\n";
+usage += "• c adds a ceiling\n";
+usage += "• e places air blocks\n";
+usage += "• a places air blocks only\n";
+usage += "• v creates a vertical maze\n";
```

Comment:
Now you have both a broken U+0095 control character and the bullet point in there.
Also, you have an UTF-8 BOM in there that wasn't there before.
I suggest you use Notepad++ to edit this and go to View -> Show Symbols -> All characters to show the broken characters.

Reply:
Also, make sure you go to Encoding -> Convert to UTF-8 (without BOM!) before you save.